src/tests/Logary.Facade.Tests/Facade.fs

```diff
@@ -62,33 +79,125 @@ let tests =
       let logger = Targets.create level
       Message.event Verbose "Hi {name}" |> logger.logSimple
 
-    testCase "extract matches" <| fun _ ->
-      let str = "Hi {ho}, we're { something going on  } special"
```

Comment:
sorry not sure what you mean on this one, do you mean the `{ something going on  }` property?